src/validators/typed_dict.rs

```diff
@@ -211,6 +211,13 @@ impl Validator for TypedDictValidator {
                             // extra logic either way
                             used_keys.insert(used_key);
                         }
+                        if *skip_validating_this_field {
+                            output_dict.set_item(&field.name_py, value)?;
+                            if let Some(ref mut fs) = fields_set_vec {
+                                fs.push(field.name_py.clone_ref(py));
+                            }
+                            continue
+                        }
```

Comment:
I'm not adding a test for this since it's already explicitly tested in the existing tests